easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.2-cpeGNU-22.06.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '4.1.2'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'cpeGNU', 'version': '22.06'}
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a400719b04375cd704d2ed063a50e42d268497a3dfede342986ab7a8d7e8dcf0']
+
+dependencies = [
+    ('rocm/5.1.4', EXTERNAL_MODULE)
+]
+
+configopts = '--with-ofi=/opt/cray/libfabric/1.15.0.0 --with-pmi=/usr'
```

Comment:
@alazzaro it would be great if HPE could add an extra environment variable to the libfabric module. Something like `CRAY_LIBFABRIC_ROOT` or `CRAY_LIBFABRIC_PREFIX` to avoid hard-coded path. At the moment, we have to update every easyconfigs that reference libfabric for every update of the CPE which is not really practical.

Reply:
The appropriate way of finding that path should probably be by querying `pkg-config libfabric`

Reply:
Indeed! That's an other way to do it